app/views/admin/adjustments/index.html.erb

```diff
@@ -42,3 +41,19 @@
     </table>
   </div>
 </div>
+
+<script>
+  $(document).ready(function () {
+      // will call refreshPartial every 15 seconds
+      setInterval(refreshPartial, 15000)
+
+  });
+
+  // calls action refreshing the partial
+  function refreshPartial() {
+    $.ajax({
+      url: '/admin/voteringar/refresh',
+      method: 'POST'
+   })
+  }
+</script>
```

Comment:
The auto refresh seems to keep running even after leaving the adjustment page. :confused: If I check in the console, I can see that `refresh` keeps running until i close the tab / enter a new address.

Also, I think we should add auto refresh to the `vote_user` page